### PR TITLE
feat: added navbar component

### PR DESCRIPTION
**Pull request by Prashantrajput11**
Added navbar 

image

### CODE REVIEW

File: components/navbar.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+.nav-container {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+	background-color: var(--primary-color);
+	font-size: 2.4rem;
+	padding: 1.6rem 0;
+	margin: 0 1.6rem;
+	box-shadow: 2px 2px 8px var(--primary-color);
```
Comment (srishti-maurya) [inline components/navbar.css:9]: if you are using this `box-shadow` color multiple times, you may store it in a variable.

**Review by srishti-maurya — APPROVED** [review]
Looks clean and good

**Review by Ayesha-7 — APPROVED** [review]
good to merge.

**Review by msdeshmukh009 — APPROVED** [review]
I think @srishti-maurya has a good suggestion apart from that I think it's good to merge.

**Pull request merged**